packages/nodes/src/ethereum.node.service.ts

```diff
@@ -0,0 +1,105 @@
+import { Inject, Injectable } from '@nestjs/common';
+import * as ganache from 'ganache';
+import { Server } from 'ganache';
+import {
+  Request,
+  APIRequestContext,
+  APIResponse,
+  BrowserContext,
+  Page,
+} from 'playwright';
+import { utils } from 'ethers';
+import { OPTIONS, ServiceUnreachableError } from './node.constants';
+
+@Injectable()
+export class EthereumNodeService {
+  readonly PORT = 7545;
+  readonly nodeUrl = `http://127.0.0.1:${this.PORT}`;
+  public static readonly defaultBalance = 6700;
+  public static readonly stakeAmount = '100';
+
+  state: { node: Server; account: string; secretKey: string } | undefined;
+
+  constructor(@Inject(OPTIONS) private options: { rpcUrl: string }) {}
+
+  async startNode() {
+    if (this.state !== undefined) return;
+    const node = ganache.server({
+      chainId: 0x1,
```

Comment:
In future, it will be great to make it configurable

Reply:
done